bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/icon/FontAwesome5CDNCSSReference.java

```diff
@@ -11,14 +11,14 @@
  *  &lt;dependency&gt;<br/>
  *      &lt;groupId&gt;org.webjars&lt;/groupId&gt;<br/>
  *      &lt;artifactId&gt;font-awesome&lt;/artifactId&gt;<br/>
- *      &lt;version&gt;5.15.4&lt;/version&gt;<br/>
+ *      &lt;version&gt;6.0.0&lt;/version&gt;<br/>
  *  &lt;/dependency&gt;<br/>
  *
- * reference for font awesome 5.x css via CDN
+ * reference for font awesome 6.x css via CDN
  */
 public class FontAwesome5CDNCSSReference extends UrlResourceReference{
```

Comment:
FontAwesome**5**
@ursjoss What do you think about this PR ?
Do we need separate FontAwesome**6** classes ?

Reply:
@martin-g I'd have to look into it a bit more, as I have not checked out the changes from 5.x to 6.x at all yet.  I suggest to remove this version bump from this PR and handle font-awsome 5.x-6.x in the context of a dedicated ticket.

Reply:
I agree!
@solomax Let's upgrade FA in a separate PR! The rest of the changes look good!

Reply:
I thought such change would be OK in this particular branch :)

Reply:
@solomax Most probably it would be, but we'd have to pull along the representations of the icons. See e.g. #923.

Reply:
sure, will try to update PR ASAP :)